<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Tracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>6</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>hh:mm:ss</string>
    </property>
    <property name="time">
     <time>
      <hour>0</hour>
      <minute>5</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> H : M : S : MS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Max AFK Time:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="afkBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string notr="true">%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>95</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Max AFK Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AFK Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>145</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Online Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>170</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PC Uprime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>95</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00:000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00:000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>145</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00:000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>84</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00:000</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>206</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
